.circleci/config.yml

```diff
@@ -146,7 +146,7 @@ jobs:
   test-integration-arm:
     <<: *integration-template
     docker:
-      - image: uroottest/test-image-arm:v4.4.0
+      - image: uroottest/test-image-arm:v5.0.0
```

Comment:
did you create the new docker image with this version? It seems not?

Reply:
you are right, just sorted out some permission issue w/ pushing to docker hub. Just pushed, triggering a new build.